salt/cli/batch.py

```diff
@@ -89,7 +89,7 @@ def get_bnum(self):
                 else:
                     return int(res)
             else:
-                return int(self.opts['batch'])
+                return int(float(self.opts['batch']))
```

Comment:
It's possible for a string (or other type) of "X.0" to be passed where int() can only handle X unless the type is cast to a float. This isn't a "bug" but is a potential issue in edge cases.